### PR TITLE
Update: November 2023 Edition

### PR DESCRIPTION
**Pull request by mrcnski**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,12 +19,12 @@ futures = "0.3.28"
 subxt = "0.32.1"
 tokio = { version = "1", features = ["full", "rt-multi-thread"] }
 
-parity-scale-codec = "3.6.8" # required by subxt
+parity-scale-codec = "3.6.5" # required by subxt
```
Comment (mrcnski) [inline Cargo.toml:22]: `3.6.8` couldn't be resolved. 😧 Neither could `3.6.8-alpha.1`.

File: Cargo.toml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 name = "pvf-checker"
 version = "0.99.0+master"
 edition = "2021"
+default-run = "pvf-checker"
```
Comment (mrcnski) [inline Cargo.toml:5]: `cargo run` by itself no longer works with multiple binaries.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ This tool uses [`subxt`](https://github.com/paritytech/subxt) to connect to an R
 ### Example
 
 ```bash
+cargo build --release
```
Comment (mrcnski) [inline README.md:14]: Even with `default-run`, you still have to run `cargo build` manually with multiple binaries. See https://github.com/rust-lang/cargo/issues/7387.

File: Cargo.toml (hunk 1 above)
Comment (ordian) [reply Cargo.toml:22]: It was yanked it seems: https://crates.io/crates/parity-scale-codec/versions.

**Review by ordian — APPROVED** [review]
Thanks!

**Comment by mrcnski** [comment]
No problem @ordian! There's also this in case you missed it:
https://github.com/ordian/kuddelmuddel/pull/4

**Pull request merged**